eslint.config.js

```diff
@@ -31,16 +31,15 @@ const eslintPluginTestsRecommendedConfig =
 //-----------------------------------------------------------------------------
 
 export default defineConfig([
-	globalIgnores(["**/tests/fixtures/", "**/dist/", "test.css"]),
+	globalIgnores(["**/tests/fixtures/", "**/dist/", "test.css", "coverage/"]),
```

Comment:
```suggestion
	globalIgnores(["**/tests/fixtures/", "**/dist/", "test.css", "coverage/", "src/build/"]),
```

I've noticed that the `src/build/` directory is also ignored in the JSON pull request.

Would it make sense to add it here as well?

https://github.com/eslint/json/pull/127

<img width="202" height="121" alt="Image" src="https://github.com/user-attachments/assets/40e8c8ad-3556-491a-a87c-6fb30f8c8c82" />